src/wallet/wallet.h

```diff
@@ -177,28 +178,33 @@ class CMerkleTx : public CTransaction
      */
     int nIndex;
 
-    CMerkleTx()
+    template<typename... Args>
+    CMerkleTx(Args&&... args)
     {
+        SetTx(std::forward<Args>(args)...);
         Init();
     }
 
-    CMerkleTx(const CTransaction& txIn) : CTransaction(txIn)
-    {
-        Init();
-    }
+    operator const CTransaction&() const { return *tx; }
```

Comment:
Where is this operator used? CTransactionRef tx is public.
Maybe a getter and making it protected would be better?